src/Files.App/Data/Items/ListedItem.cs

```diff
@@ -268,6 +270,31 @@ public string FileSize
 
 		public long FileSizeBytes { get; set; }
 
+		private int imageWidth;
+		public int ImageWidth
+		{
+			get => imageWidth;
+			set
+			{
+				SetProperty(ref imageWidth, value);
+				OnPropertyChanged(nameof(DimensionsDisplay));
+			}
+		}
+		
+		private int imageHeight;
+		
+		public int ImageHeight
+		{
+			get => imageHeight;
+			set
+			{
+				SetProperty(ref imageHeight, value);
+				OnPropertyChanged(nameof(DimensionsDisplay));
+			}
+		}
+
+		public string DimensionsDisplay => IsImage ? $"{ImageWidth} x {ImageHeight}" : string.Empty;
```

Comment:
I see many apps use x instead of symbols.
Is there a reason why you suggested?

Reply:
> I see many apps use x instead of symbols.
> Is there a reason why you suggested?

Just because somebody else does it, doesn't mean it's right 😉

Please use
![image](https://github.com/files-community/Files/assets/65828559/81d58dc0-886b-45fe-b53c-ff506bcc5df8)
![image](https://github.com/files-community/Files/assets/65828559/cdd3b9f7-a462-4291-ad96-2c3086911b27)

Reply:
It sounds right.

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/m/multiplication-sign